README_ja-JP.md

```
@@ -0,0 +1,335 @@
+```
+
+以下を行う:
+
+1. 両方のAPKファイルをデコンパイル
+2. 変更されたファイルに対応する差分パッチを生成
+3. 新しいファイルをパッチディレクトリにコピー
+4. 削除リストと追加リストを作成
+
+### パッチディレクトリの構造
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)